### PR TITLE
user quality of life improvements 

### PR DESCRIPTION
**Pull request by Maarten-vd-Sande**
**What problem is the PR solving / What's new?**
Some minor QOL stuff. If something fails, redirect to docs:
image

Moved `get_genome` and `get_genome_annotation` to scripts, so when genome providers are offline, the error message changes from 
```
Error in rule get_genome:
    jobid: 0
    output: /scratch/tschaefers/10x_MvanBergen/genomes/hg38/hg38.fa
    log: /scratch/tschaefers/10x_MvanBergen/results/log/get_genome/hg38.genome.log (check log file(s) for error message)
RuleException:
ValueError in line 32 of /vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/seq2science/rules/get_genome.smk:
Unknown provider
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/snakemake/executors/__init__.py", line 2163, in run_wrapper
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/seq2science/rules/get_genome.smk", line 32, in __rule_get_genome
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/genomepy/provider.py", line 88, in create
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/concurrent/futures/thread.py", line 56, in run
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/vol/mbconda/tschaefers/envs/seq2science/lib/python3.6/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Exiting because a job execution failed. Look above for error message
```

To the slightly friendlier image above :point_up: 

I also added an example (with images) how to make a PR to change the docs. Maybe we can send this to some of our wet-lab colleagues and they can also take responsibility, if they want, for the docs.

**Checklist**
- [x] I made a PR to develop (not master)
- [x] If applicable: I updated the docs
- [x] I updated the CHANGELOG
- [x] These changes are covered by the tests

### CODE REVIEW

**Review by siebrenf — APPROVED** [review]
The error message is a bit bulky, but more friendly than the wall of red :) Maybe it should also refer to the log file of the erroneous rules before mentioning the FAQ?

**Comment by Maarten-vd-Sande** [comment]
I don't know how to refer to the log files of the failed rules. The first question in the FAQ is about checking the log though?

File: seq2science/rules/configuration_logging.smk [hunk 1]
```diff
@@ -10,10 +13,35 @@ onstart:
...
+ /  \  /  \  /  \(  _ \/ ___)/ \  
+(  O )(  O )(  O )) __/\___ \\\\_/  
+ \__/  \__/  \__/(__)  (____/(_)  
+
+
+One or more rules did not finish as expected!
+
+
+Please take a look at our Frequently Asked Questions for help: 
+https://vanheeringen-lab.github.io/seq2science/content/faq.html
```
Comment (siebrenf) [inline seq2science/rules/configuration_logging.smk:36]: I was thinking something like

> One or more rules did not finish as expected!
> 
> 
> Please take a look at the log files of the failed rule(s), and our Frequently Asked Questions: 
https://vanheeringen-lab.github.io/seq2science/content/faq.html

to save users the extra click. Didn't see the faq describes this immediately, so we could skip it :)

**Comment by Maarten-vd-Sande** [comment]
I added your proposed sentence

**Pull request merged**